src/webdrivers/chrome/install_driver_executor.cr

```diff
@@ -14,7 +14,7 @@ class Webdrivers::Chrome::InstallDriverExecutor
 
     FileUtils.cd(driver_directory) do
       zip = download_file(from: download_url, to: download_url_filename)
-      Common::ZipExtractor.new(zip, driver_name, driver_directory).extract
+      Common::ZipExtractor.new(zip, File.join("chromedriver-#{download_url_platform}", driver_name), driver_directory).extract
```

Comment:
The zip contents used to just be `chromedriver.zip/chromedriver` but now it's `chromedriver.zip/chromedriver-linux64/chromedriver`